generator/class.go

```diff
@@ -82,6 +79,11 @@ func generateDartClass(outputFile io.Writer, ts *types.TypeName, st *types.Struc
 		panic(fmt.Sprintf("expected StructFormatter, got %T", registry.GetTypeFormatter(ts.Type())))
 	}
 
+	fields := extractFields(st)
+
+	if len(fields) > 0 {
```

Comment:
Can't add it to a type wo/ fields (i.e. Empty), it causes the build runner to fail.